oneanddone/functional_tests/pages/base.py

```diff
@@ -0,0 +1,60 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Are these tests simply trnasplanted from oneanddone-tests? I'm just wondering how detailed you want them reviewed.